azure-pipelines.yml

```
@@ -0,0 +1,44 @@
+variables:
+  CI: true
+
+jobs:
+  - job: package_selection
+    displayName: 'Package selection'
+    pool:
+      vmImage: 'ubuntu latest'  
+    steps:
+      - task: Bash@3
+        displayName: 'Determine packages to be build'
+        name: matrix
+        inputs:
+          targetType: 'inline'
+          script: |
+            wget https://raw.githubusercontent.com/tue-robotics/tue-env/feature/azure_scripts/ci/packages.sh
```

```suggestion
            wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/packages.sh
```
Need to be committed after merging tue-robotics/tue-env#417